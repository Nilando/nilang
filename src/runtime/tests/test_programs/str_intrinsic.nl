print(str(""));
print(str(null));
print(str(0));
print(str(1.234));
print(str(true));
print(str(false));
print(str("test"));
print(str($foo));
