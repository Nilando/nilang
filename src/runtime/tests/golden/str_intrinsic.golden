print(str(""));
print(str(null));
print(str(0));
print(str(1.234));
print(str(true));
print(str(false));
print(str("test"));

%%%%

0
1.234
true
false
test
